packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -268,6 +291,24 @@ export default class TooltipAnchor extends React.Component<Props, State>
     };
 
     render() {
-        return this.props.children(this.state.active);
+        // We need to make sure we can anchor on our content.
+        // If the content is just a string, we wrap it in a Text element
+        // so as not to affect styling or layout but still have an element
+        // to anchor to.
+        const {children, ids} = this.props;
+        const anchorableChildren =
+            typeof children === "string" ? (
+                <WBText>{children}</WBText>
+            ) : (
+                children
+            );
+
+        if (ids) {
+            return React.cloneElement(anchorableChildren, {
+                "aria-describedby": ids.get(TooltipAnchor.ariaContentId),
+            });
+        } else {
+            return anchorableChildren;
+        }
```

Comment:
Again, I think this should be a method like:

```
return this._renderAccessibleChildren()
```

I think this helps make the code self-documenting without needing maintainers to read and understand every line.